rust/main/agents/relayer/src/msg/metadata/base.rs

```diff
@@ -208,11 +208,20 @@ pub enum IsmCachePolicy {
     IsmSpecific,
 }
 
+#[derive(Debug, Clone, Default, Deserialize, PartialEq)]
+#[serde(tag = "type", content = "data", rename_all = "camelCase")]
```

Comment:
I found the `data` naming of the content a bit confusing, how about calling it `context`

Reply:
agreed, good call